scripts/start_gaudi_vllm_server.sh

```diff
@@ -38,11 +39,11 @@ module_ids=None
 host=127.0.0.1
 port=30001
 dtype=bfloat16
-input_range=(4 1024)
+input_range=(4 16384)
 output_range=(4 2048)
-max_num_batched_tokens=8192
-max_model_len=4096
-max_num_seqs=128
+max_num_batched_tokens=20480
+max_model_len=20480
+max_num_seqs=64
```

Comment:
Those are the default values, please keep it unchanged and set the actual value by passing the corresponding parameters.